packages/loopring_v3/contracts/aux/bridge/Bridge.sol

```diff
@@ -243,27 +244,24 @@ contract Bridge is IBridge, ReentrancyGuard, Claimable
         }
     }
 
-    function setConnectorTrusted(
+    function trustConnector(
         address connector,
         bool    trusted
         )
         external
-        onlyOwner
+        onlyFromExchangeOwner
```

Comment:
This won't work because the exchange owner is a `SelectorBasedAccessManager` which can only target a single contract which needs to be the exchange. So to be able to be the owner of all these converters we would have to implement extra functionality to be able to do so, which I don't think makes sense. Only our multisig should be able to call this function.

Reply:
Thank you for pointing this out. I'll revert the change.